lib/pages/school_gallery/school_gallery_page.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:school_erp/pages/school_gallery/widgets/school_gallery_card.dart'; // Update this to the appropriate path for the gallery widget
+import 'package:school_erp/pages/common_widgets/default_layout.dart';
+
+class SchoolGalleryPage extends StatefulWidget {
+  const SchoolGalleryPage({super.key});
+
+  @override
+  _SchoolGalleryPageState createState() => _SchoolGalleryPageState();
+}
+
+class _SchoolGalleryPageState extends State<SchoolGalleryPage> {
+  @override
+  Widget build(BuildContext context) {
+    return const DefaultLayout(
+        title: "School Gallery", content: [SchoolGalleryCard()]);
+  }
+}
```

Comment:
why not stateless?

Reply:
I just based it on the recent refactors. tho it can be a stateless widget. I also don't know the answer to that question.

Reply:
if we are not managing states and thus could get away by using stateless widgets
then its preferrable as 

1) stateless has less boilerplate and is more simple
2) a stateful object would consume more resource compared to a stateless widget because of the state object that is being created